### PR TITLE
add Levinson keymap

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Rossman360** [comment]
oops, I may have forgotten to fetch from master before pushing to dev branch, thus the tragicforce commits.

**Comment by noroadsleft** [comment]
```
git fetch upstream master
git rebase upstream/master
git push --force-with-lease
```

should fix that for you.

**Comment by Rossman360** [comment]
Ok, I think I rebased branches successfully.

**Comment by noroadsleft** [comment]
You did, but I gave you bad instructions. :man_facepalming: 

1. Run `git rebase -i upstream/master`
   Should give you a window similar to this:
   image
2. Type `:1,5s/pick/drop/g` then hit Enter
   The word `pick` on the first five lines should change to `drop`.
3. Type `:wq` and hit Enter
   That should kick you back into the Terminal. Git should do some processing and then return something like:
   `Successfully rebased and updated dev_branch.`
4. Run `git push --force-with-lease`
   That should remove the old commits from your branch and update the PR again.

**Comment by Rossman360** [comment]
my terminal didn't let me do any of that fancy text replacing, but I got the unneeded commits dropped. Thanks!

**Comment by noroadsleft** [comment]
Your terminal might be configured to use a different editor. I thought Vim was the default but I could be wrong. Glad you got it figured out!

File: keyboards/keebio/levinson/keymaps/rossman360/keymap.c [hunk 1]
```diff
@@ -0,0 +1,137 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/rossman360/keymap.c]: You don't need Line 3 at all.

File: keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c~HEAD [hunk 2]
```diff
@@ -0,0 +1,145 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c~HEAD]: `git rm -- keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c~HEAD`

**Pull request merged**